<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Random Movie Picker</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="1" column="0">
         <widget class="QWidget" name="widget" native="true">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Directorios de películas:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPlainTextEdit" name="movieListEditText">
             <property name="placeholderText">
              <string>Pega aquí los directorios (ruta completa) donde tienes películas, todos deben estar separados por un salto de línea</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="doItButton">
          <property name="text">
           <string>SELECCIONAR PELÍCULA AL AZAR</string>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="lastMovieLabel">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Última selección: -</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="styleSheet">
        <string notr="true">QWidget #page_2{
	
	background-color: rgb(0, 0, 0);
}</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>21</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>AHORA VERÁS</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <widget class="QWidget" name="widget_3" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QWidget #widget_3{
	border-radius: 10px;
	border: 1px solid black;
	background-color: rgb(255, 255, 255);
}s</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <property name="sizeConstraint">
               <enum>QLayout::SetMaximumSize</enum>
              </property>
              <item row="0" column="1">
               <widget class="QLabel" name="imageLabel">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>SIN IMAGEN</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QWidget" name="widget_5" native="true">
                <layout class="QVBoxLayout" name="verticalLayout">
                 <property name="sizeConstraint">
                  <enum>QLayout::SetMinimumSize</enum>
                 </property>
                 <item>
                  <widget class="QPushButton" name="openFileButton">
                   <property name="text">
                    <string>Abrir archivo</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="openFolderButton">
                   <property name="text">
                    <string>Abrir carpeta</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="movieFileLineEdit">
                   <property name="font">
                    <font>
                     <family>Calibri</family>
                     <pointsize>12</pointsize>
                     <weight>75</weight>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>Sin título</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="qualityLabel">
                   <property name="font">
                    <font>
                     <family>Calibri</family>
                     <pointsize>12</pointsize>
                     <italic>true</italic>
                    </font>
                   </property>
                   <property name="text">
                    <string>Sin datos</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                   <property name="wordWrap">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>560</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfigurar">
    <property name="title">
     <string>Selector</string>
    </property>
    <addaction name="actionConfigurar"/>
   </widget>
   <addaction name="menuConfigurar"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConfigurar">
   <property name="text">
    <string>Configurar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
